system/datatypes/peer_reviewed_article.php

```diff
@@ -757,7 +758,7 @@ public function discussionAction($type) {
 			$form->validateEmailField(self::FIELD_NAME_DISCUSSION_COMMENTER_EMAIL);
 		}
 		$form->validateRequiredField(self::FIELD_NAME_DISCUSSION_COMMENT . '/new_' . $type);
-
+		$form->validateRequiredField(self::FIELD_NAME_DISCUSSION_SUBJECT);
```

Comment:
Did you intend to remove the empty line here? It seems useful to keep it, to separate the validation from the rest of the processing.